PokeAppAndroid/Resources/layout/activity_pokemon_home.xml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent">
+
+    <!-- The Main Content View-->
+
+    <FrameLayout
```

Comment:
There is no documentation about that, i guess that maybe we can use it but it will need a deep research because i found a lot of deprecated code so maybe FragmentContainerView is not supported, and this was the way that works similar to FragmentContainerView